.github/workflows/release.yml

```diff
@@ -142,18 +142,34 @@ jobs:
         id: create-github-release
         if: ${{ ! inputs.skip-github-release }}
         run: |
+          echo "### GitHub Release" >> "$GITHUB_STEP_SUMMARY"
           IFS=$'\n' artifacts_array=($ARTIFACTS)
+          echo "#### Attaching Release Artifacts" >> "$GITHUB_STEP_SUMMARY"
           for file in "${artifacts_array[@]}";
           do
-              echo "Attaching file '$file'"
+              echo "Attaching artifact '$file'"
+              echo "* \`$file\`" >> "$GITHUB_STEP_SUMMARY"
           done
-          release_url=$(gh release create --draft --latest --title "$TITLE" --notes "$NOTES" --target main $TAG "${artifacts_array[@]}")
-          echo "Created release $TAG with title '$TITLE' at $release_url" >> "$GITHUB_STEP_SUMMARY"
+          echo "" >> "$GITHUB_STEP_SUMMARY"
+          release_url=$(gh release create --latest --title "$TITLE" --notes "$NOTES" --target main $TAG "${artifacts_array[@]}")
+          echo "Created release $TAG with title '$TITLE' at $release_url ✅" >> "$GITHUB_STEP_SUMMARY"
           echo "release-url=$release_url" >> "$GITHUB_OUTPUT"
+
+          echo "#### Creating Additional Tags" >> "$GITHUB_STEP_SUMMARY"
```

Comment:
Maybe a comment or a reference to requirements / design would be nice, here? 

E.g.
* `# Go projects need an additional tag prefixed with "v"` 
* or `[impl->dsn~release-workflow.create-golang-tags~1]`

Reply:
Oh, I see an additional challenge - maybe 2 challenges: 
* file might be generated by PK so the reqs might apply to the template file but should not affect req tracing of a project using PK to generate its workflow files. :thinking: Oh oh .... 
* maybe file is even modified as PK itself has deviates from default?